tests/Microsoft.Fx.Portability.MetadataReader.Tests/ManagedMetadataReaderTests.cs

```diff
@@ -113,10 +115,15 @@ private void CompareSpecificDependency(string path, string v)
 
             var dependencies = dependencyFinder.FindDependencies(new[] { assemblyToTestFileInfo }, progressReporter);
 
-            var found = dependencies.Dependencies
-                .Any(d => string.Equals(d.Key.MemberDocId, v, StringComparison.Ordinal));
+            foreach (var dependency in dependencies.Dependencies)
+            {
+                if (string.Equals(dependency.Key.MemberDocId, v, StringComparison.Ordinal))
+                {
+                    return;
+                }
+            }
```

Comment:
I think I did it for debugging purposes - I kept changing it to this to identify what the dependencies are.  FIgured it would be better to keep it.